config/locales/jobseekers.yml

```diff
@@ -43,6 +43,11 @@ en:
         page_title: Saved jobs
     subscriptions:
       index:
+        heading_actions: Actions
```

Comment:
If I was being super anal I might say we could nest these like
```
heading:
  actions: Actions
``` 
etc, but in reality I don't mind too much